docs/search-configuration/custom-dictionaries.md

```diff
@@ -56,18 +56,61 @@ words
 ## Japanese User Dictionary
 Your site's language must be set to Japanese to see this option.
 
-A user dictionary provides a way to control how words are broken up when searching. If there are compound words or phrases specific to this site, such as the names of authors or celebrities that users may search for, they can be specified here to increase search relevancy.
+A user dictionary provides a way to control how words are broken up when searching. If there are compound words or phrases specific to your site, such as the names of imaginary places, or authors and celebrities that users may search for, they can be specified here to increase search relevancy.
 
-The syntax for the provided text file should follow the CSV format:
+The syntax for the provided text file should follow the Comma-Separated Values (CSV) format:
 
 ```csv
-text, token 1 ... token n, reading 1 ... reading n, part-of-speech tag
+text,tokens,readings,part-of-speech
 ```
 
-For example:
+1. `text` is the compound word or phrase that appears in your content, such as a name
```

Comment:
should we indicate that `text` should only contain one word?

Reply:
Could do, that's what I'm referring to with compound. Maybe it'll make more sense once it's been translated but I think for now lets return to this in beta 2 or the RC